examples/engine/3_responses.py

```diff
@@ -59,57 +62,66 @@ def cannot_talk_about_topic_response(ctx: Context, actor: Actor, *args, **kwargs
 
 
 def fallback_trace_response(ctx: Context, actor: Actor, *args, **kwargs) -> Any:
-    logger.warning(f"ctx={ctx}")
     return {"previous_node": list(ctx.labels.values())[-2], "last_request": ctx.last_request}
 
-
+# %%
 toy_script = {
     "greeting_flow": {
-        "start_node": {  # This is an initial node, it doesn't need an `RESPONSE`
+        "start_node": {  # This is an initial node, it doesn't need a `RESPONSE`.
             RESPONSE: "",
-            TRANSITIONS: {"node1": cnd.exact_match("Hi")},  # If "Hi" == request of user then we make the transition
+            TRANSITIONS: {"node1": cnd.exact_match("Hi")}, 
+            # If "Hi" == request of user then we make the transition
         },
         "node1": {
-            RESPONSE: rsp.choice(["Hi, what is up?", "Hello, how are you?"]),  # random choice from candicate list
-            TRANSITIONS: {"node2": cnd.exact_match("i'm fine, how are you?")},
+            RESPONSE: rsp.choice(["Hi, what is up?", "Hello, how are you?"]),
+            # Random choice from candicate list.
+            TRANSITIONS: {"node2": cnd.exact_match("I'm fine, how are you?")},
         },
         "node2": {
             RESPONSE: "Good. What do you want to talk about?",
             TRANSITIONS: {"node3": cnd.exact_match("Let's talk about music.")},
         },
-        "node3": {RESPONSE: cannot_talk_about_topic_response, TRANSITIONS: {"node4": cnd.exact_match("Ok, goodbye.")}},
-        "node4": {RESPONSE: upper_case_response("bye"), TRANSITIONS: {"node1": cnd.exact_match("Hi")}},
```

Comment:
Do we need add limits for examples max line length 80 chars? It will be more easy to check

Reply:
80 chars is far too few. In my opinion, this will for sure lead to unnecessary code betweem lines splitting.